CONTRIBUTING.md

```diff
@@ -28,21 +44,22 @@ If you have the package manager installed, you can install pnpm using the follow
 brew install pnpm
 ```
 
-To develop Tres core or any of the packages, run `pnpm install` in the root of the project. This will install all the dependencies and link the packages together. There is also a `postinstall` script that will build all the packages.
 
 ## Development
 
-TresJS is an ecosystem of packages, each one of them has its own purpose. The main package is `@tresjs/core` which is the core of the library. The other packages are plugins that extend the core functionality, like `@tresjs/cientos` which is a plugin that adds a bunch abstractions and composables to make it easier to create 3D scenes.
+To start developing, you can run `pnpm run playground` in the root folder.
+
+This will start the dev server for the playground `http://localhost:5174/` where you can test the changes you are making on the `src` folder. 
```

Comment:
```suggestion
This will start the dev server for the playground at `http://localhost:5173/` where you can test the changes you are making in the `src` folder. 
```